### DISCUSSION TITLE
Translate to danish

### DISCUSSION THREAD

**Discussion by garuda1954** [elementary/triage#178] [Q&A] [answered]
Hi
I want to translate website and programs, but when I try to translate on elementary Weblate I can't save the translation. My email on Weblate is carl at bluemarlin.dk and my username is Snubbi

**Comment by 1280px** [comment, marked as answer]
Hello! In order to be able to save translations, you need to get a permission in elementary l10n chat:

https://elementary.io/get-involved#translations (click "Join Chat" button)
